sdk/quantum/quantum-jobs/recordings/node/quantum_job_lifecycle/recording_test_quantum_job_lifecycle.js

```diff
@@ -1,18 +1,16 @@
 let nock = require('nock');
 
-module.exports.hash = "d72be875b213407d3bd89ae5a3328135";
+module.exports.hash = "ab4dd1c6afdb4de9db1250bebba8700d";
 
-module.exports.testInfo = {"uniqueName":{"input-":"input-161240060833601459","job-":"job-161240060889002218","jobname-":"jobname-161240060889009123"},"newDate":{}}
+module.exports.testInfo = {"uniqueName":{"input-":"input-162395518609708661","job-":"job-162395518637801404","jobname-":"jobname-162395518637800407"},"newDate":{}}
 
 nock('https://login.microsoftonline.com:443', {"encodedQueryParams":true})
-  .post('/88888888-8888-8888-8888-888888888888/oauth2/v2.0/token', "response_type=token&grant_type=client_credentials&client_id=ce7bd34e-0000-0000-0000-000000000000&client_secret=clientsecret&scope=https%3A%2F%2Fquantum.microsoft.com%2F.default")
+  .post('/12345678-1234-1234-1234-123456789012/oauth2/v2.0/token', "response_type=token&grant_type=client_credentials&client_id=ce7bd34e-0000-0000-0000-000000000000&client_secret=clientsecret&scope=https%3A%2F%2Fsanitized%2F")
```

Comment:
Is that `sanitized` being done by MSAL? @sadasant

Reply:
no, that’s done by the recorder, cc @HarshaNalluru

Reply:
Ok that looks good then